framework-examples/react/src/Checkbox.js

```diff
@@ -101,6 +110,45 @@ export default class Checkbox extends PureComponent {
     isAttachedToDOM: () => Boolean(this.refs.nativeCb),
   });
 
+  // For browser compatibility we extend the default adapter which checks for css variable support.
+  rippleFoundation = new MDCRippleFoundation(Object.assign(MDCRipple.createAdapter(this), {
+    isUnbounded: () => true,
+    isSurfaceActive: () => this.refs.root[MATCHES](':active'),
```

Comment:
what you actually want to check is that the _input_ element is active, rather than the root. See https://github.com/material-components/material-components-web/pull/241